backend/mora/cli/base.py

```diff
@@ -251,8 +251,16 @@ def expand_suite(suite):
 
 
 @cli.command()
-@click.option('--simple', is_flag=True, help='Run with the simple fixtures.')
-def full_run(simple):
+@click.option('--fixture', type=click.Choice(['minimal', 'simple', 'dummy',
+                                              'small', 'normal', 'large']),
+              default='dummy',
+              help='Choose the initial dataset.')
+@click.option('--dump-to', type=click.File(mode='w', lazy=True, atomic=True),
+              help='Dump the database to the given file.')
+@click.option('--backend-only', is_flag=True,
+              help="Don't run the ``vue-cli-service`` server for frontend "
+              "development.")
+def full_run(fixture, dump_to, backend_only):
```

Comment:
This function has a lot going on now. 150 lines of multiple if/else structures. I'd suggest a refactor into smaller, more manageable chunks.